jenkinsfile-runner-prepackaged/Dockerfile.build

```diff
@@ -0,0 +1,25 @@
+FROM jenkins/jenkinsfile-runner:adoptopenjdk-11-hotspot
```

Comment:
FTR https://github.com/jenkinsci/jenkinsfile-runner/pkgs/container/jenkinsfile-runner can be used for the latest version of Jenkinsfile Runner